docs/guides/experimental-exporter.md

```diff
@@ -61,3 +61,81 @@ Services are just logical groups of resources used for filtering and organizatio
+| [databricks_mws_workspaces](docs/resources/mws_workspaces.md) | No |
+| [databricks_node_type](docs/data-sources/node_type.md) data | No |
+| [databricks_notebook](docs/resources/notebook.md) | Yes |
+| [databricks_notebook](docs/data-sources/notebook.md) data | Yes |
+| [databricks_notebook_paths](docs/data-sources/notebook_paths.md) data | No |
+| [databricks_obo_token](docs/resources/obo_token.md) | No |
+| [databricks_permissions (job)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (cluster)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (instance_pool)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (cluster_policy)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (repo)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (token)](docs/resources/permissions.md) | No |
+| [databricks_permissions (password)](docs/resources/permissions.md) | No |
+| [databricks_permissions (delta live tables pipeline)](docs/resources/permissions.md) | No |
+| [databricks_permissions (notebook)](docs/resources/permissions.md) | No |
+| [databricks_permissions (directory)](docs/resources/permissions.md) | No |
+| [databricks_permissions (mlflow experiment)](docs/resources/permissions.md) | No |
+| [databricks_permissions (mlflow registered model)](docs/resources/permissions.md) | No |
+| [databricks_permissions (sql_endpoint) ](docs/resources/permissions.md) | No |
+| [databricks_pipeline](docs/resources/pipeline.md) | No |
+| [databricks_repo](docs/resources/repo.md) | Yes |
+| [databricks_schema](docs/resources/schema.md) | No |
+| [databricks_schemas](docs/data-sources/schema.md) data | No |
+| [databricks_secret](docs/resources/secret.md) | Yes |
+| [databricks_secret_acl](docs/resources/secret_acl.md) | Yes |
+| [databricks_secret_scope](docs/resources/secret_scope.md) | Yes |
+| [databricks_spark_version](docs/data-sources/spark_version.md) data | No |
+| [databricks_sql_dashboard](docs/resources/sql_dashboard.md) | No |
+| [databricks_sql_endpoint](docs/resources/sql_endpoint.md) | No |
+| [databricks_sql_global_config](docs/resources/sql_global_config.md) | No |
+| [databricks_sql_permissions](docs/resources/sql_permissions.md) | No |
+| [databricks_sql_query](docs/resources/sql_query.md) | No |
+| [databricks_sql_visualization](docs/resources/sql_visualization.md) | No |
+| [databricks_sql_widget](docs/resources/sql_widget.md) | No |
+| [databricks_storage_credential](docs/resources/storage_credential.md) | No |
+| [databricks_table](docs/resources/table.md) | No |
+| [databricks_tables](docs/data-sources/table.md) data | No |
+| [databricks_token](docs/resources/token.md) | No |
+| [databricks_user](docs/resources/user.md) | Yes |
+| [databricks_user_instance_profile](docs/resources/user_instance_profile.md) | No (Depricated) |
+| [databricks_workspace_conf](docs/resources/workspace_conf.md) | No |
```

Comment:
we expose it: https://github.com/databrickslabs/terraform-provider-databricks/blob/master/exporter/importables.go#L838